### PR TITLE
Add abs() variants.

### PR DESCRIPTION
**Pull request by aiw-google**
The stdlib abs() function has an unexpected behavior: abs(INT_MIN) returns INT_MIN.
This commit adds three variants of the abs() function that eliminate that behavior.

Issue:
EOP-531

### CODE REVIEW

File: include/nlabs-variants.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/*
+ *
+ *    Copyright (c) 2018 Nest Labs, Inc.
```
Comment (gerickson) [inline include/nlabs-variants.h]: This should probably be "Google LLC" at this point.

File: src/nlabs-variants.c [hunk 2]
```diff
@@ -0,0 +1,69 @@
+/*
+ *
+ *    Copyright (c) 2018 Nest Labs, Inc.
```
Comment (gerickson) [inline src/nlabs-variants.c]: This should probably be "Google LLC" at this point.

File: include/nlabs-variants.h (hunk 1 above)
Comment (aiw-google) [reply include/nlabs-variants.h]: Done.

File: src/nlabs-variants.c (hunk 2 above)
Comment (aiw-google) [reply src/nlabs-variants.c]: Done.

**Comment by gerickson** [comment]
Looks good. Before you merge, please update CHANGES, .default-version, tag it to match, push the tag and changes and then merge.

**Comment by aiw-google** [comment]
Thanks, Grant.

**Pull request merged**